fdm-app/app/routes/farm.$b_id_farm.settings.properties.tsx

```
@@ -47,19 +62,31 @@ export default function FarmSettingsPropertiesBlock() {
+                ? loaderData.farm.b_businessid_farm
+                : "",
+            b_address_farm: loaderData.farm.b_address_farm
+                ? loaderData.farm.b_address_farm
+                : "",
+            b_postalcode_farm: loaderData.farm.b_postalcode_farm
+                ? loaderData.farm.b_postalcode_farm
+                : "",
         },
     })
 
     useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: form.reset

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)